### DISCUSSION TITLE
Migrate from old registry to cloud registry

### DISCUSSION THREAD

**Discussion by Bobablocks** [WalletConnect/walletconnect-monorepo#1489] [Developer Support] [answered]
Hi, i wanted to migrate my existing registry as listed here https://github.com/WalletConnect/walletconnect-registry/issues/422 on january 2022 to my cloud registry account which has registered as [admin@bobablocks.io](mailto:admin@bobablocks.io).

To be noted, our app has changed its name and several meta data including deep links / app links, support desktop version & support multiple chains.

could someone help?  Thanks 🙏

**Comment by looter** [comment, marked as answer]
@Bobablocks done

**Reply by Bobablocks** [reply]
Thank You!
